File: Lab1/Lab1/src/main/java/Lab1/Lab1/entities/Post.java
package Lab1.Lab1.entities;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Document(collection = "post")
@AllArgsConstructor
@NoArgsConstructor
public class Post {
    @Id
    long id;
    String content;
    String title;
    String author;
}

File: lab/lab7/src/component/Post.js
const Post = (props) => {
    return (
        <div className="Content" onClick={props.onClick}>
            <h1>{props.id}</h1>
            <h3>{props.Title}</h3>
            <p>{props.Author}</p>
        </div>
    );
}

export default Post;
